app/frontend/src/features/auth/login/Login.tsx

```diff
@@ -33,18 +47,28 @@ export default function Login() {
   return (
     <>
       {authenticated && <Redirect to={redirectTo} />}
-      <Typography variant="h2">Login</Typography>
-      {error && <Alert severity="error">{error}</Alert>}
-
-      <Switch>
-        <Route path={loginPasswordRoute}>
-          <PasswordForm />
-        </Route>
-
-        <Route path={loginRoute}>
-          <UsernameForm />
-        </Route>
-      </Switch>
+      <Box className={authClasses.backgroundBlurImage}></Box>
+      <Box className={authClasses.page}>
+        <AuthHeader>Welcome back!</AuthHeader>
+        {error && <Alert severity="error">{error}</Alert>}
```

Comment:
This is not full width for some reason

Reply:
Because of `authClasses.page`

Reply:
Yes, but shouldn't it be like this? There are paddings in design

Reply:
I guess it has its own extra padding 

![Screenshot_2021-02-22-00-01-49-117_com microsoft emmx](https://user-images.githubusercontent.com/32044095/108643184-4ddfc780-74a1-11eb-9a1d-c8b7a613619b.jpg)

Reply:
Wow, I did not know we have review apps!

Reply:
Yep the CI is great, you can access all the previews for storybook and frontend from gitlab. No need to pull branches

Reply:
This is still present. It's fixed if you give the alert a class which has `width: "100%"`

Reply:
Oh, I did not know you mean error box! Fixed!